services/worker/src/worker/job_runners/config/size.py

```diff
@@ -70,6 +83,11 @@ def compute_config_size_response(dataset: str, config: str) -> ConfigSizeRespons
                 "num_bytes_memory": sum(split_size["num_bytes_memory"] for split_size in split_sizes),
                 "num_rows": sum(split_size["num_rows"] for split_size in split_sizes),
                 "num_columns": num_columns,
+                "estimated_num_rows": sum(
+                    split_size["estimated_num_rows"] or split_size["num_rows"] for split_size in split_sizes
```

Comment:
as mentioned above: it relies on `split_size["num_rows"]` being always set, even if it's `0` when we don't have the value. So: in that case, we can estimate the split has 0 rows.

I think it's OK for now, but maybe we should open an issue to handle this case better in the future.